docs/modules/ROOT/pages/how-tos/workflows/keep-remote-pipelines-up-to-date.adoc

```diff
@@ -0,0 +1,107 @@
+= Maintaining remote Tekton pipelines up to date
+
+[IMPORTANT]
+====
+* This article took for granted the reader knows what a link:https://tekton.dev/docs/pipelines/resolution/[Tekton remote pipeline] is. If this is not the case, please read more about this feature before proceeding.
+====
+
+== The advantage of using a remote pipeline
+
+This approach allows a team or an individual contributor to manage plenty of Pipelines from a single repository as source of truth. But there also other good reasons:
+
+. Pipelines are stored in a Git repository and fetched during execution, eliminating the need to maintain local or static pipeline definitions.
+. Teams can leverage Git’s versioning to access specific pipeline branches
+. In case of issues, it's very simple to rollback to a previous pipeline version, and apply it globally among the Konflux Components.
+. Reduced maintenance overhead
+
+To start referencing a remote Pipeline from a PipelineRun using a *git resolver*, replace *pipelineSpec* with *pipelineRef*:
+
+[source,yaml]
+----
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata: ... # Omitted for brevity
+spec:
+  params: ... # Omitted for brevity
+  pipelineRef:
+    resolver: git
+    params:
+      - name: org
+        value: redhat
+      - name: scmType
+        value: gitlab
+      - name: serverURL
+        value: https://gitlab.cee.redhat.com
+      - name: url
+        value: "https://gitlab.cee.redhat.com/your-group-name/your-repo-name.git"
+      - name: revision
+        value: main # Specify a branch to target
+      - name: pathInRepo
+        value: "pipelines/your-pipeline-name.yaml" # Internal repository path to your pipeline definition YAML file
+      - name: token
+        value: pipelines-as-code-secret # Name of the Secret
+      - name: tokenKey
+        value: password # Name of the attribute within your Secret, that stores the password/token
+  workspaces: ... # Omitted for brevity
+----
+
+== The drawback of having a remote pipeline
+
+Although Tekton remote pipelines is a huge improvement, it comes with a drawback; the Pipeline CRDs will not be stored in the Konflux Component repository anymore, so MintMaker will not be able to update the TaskRef digests for us.
```

Comment:
I think that this slightly misrepresents the situation. There is no pipeline CRD, but the Component CR has a reference to the specific pipeline that was used/requested when onboarding (or [reconfigured](https://github.com/konflux-ci/docs/blob/main/docs/modules/ROOT/pages/how-tos/configuring/reconfiguring-build-pipeline.adoc)).

MintMaker doesn't update the TaskRefs because anything is in the CR, but because those task references are in a "known" location.